DB_Init.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+echo "DB is being initialized"
+
+curl http://localhost:8080/admin/add_screening -H "content-type: application/json" -X POST -s -d '{
```

Comment:
czy to jest dobra ścieżka, zgodna z podejściem REST?

Reply:
Nie, fragment localhost:8080 jest napisany na sztywno. Lepszym rozwiązaniem byłoby  przynajmniej podstawienie w to miejsce zmiennej.